ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java

```diff
@@ -31,7 +31,7 @@ public void setup() {
         new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
     }
 
-    @Ignore
+	@Test
     @Category(BuildTests.class)
     public void searchActiveVisitsByPatientNameOrIdOrLastSeenTest() throws Exception {
```

Comment:
check if you really need the throws, otherwise get rid of it and get @Test at the same line as @Category

Reply:
We need to throw exception  to handle any error that might be thrown with in searching patient ids

Reply:
You obviously what the function not to throw any exception, right? If so then its better to get rid of the _throws_